docker-dev/check-status.sh

```diff
@@ -1,5 +1,3 @@
-. /tmp/iip-configuration.sh
```

Comment:
what's the purpose of this duplicate check-status.sh script? can it be removed regardless of my other comment suggesting to remove its notion altogether? Why are any of the `docker-dev` folder contents duplicated for that matter, can they just use the contents under the `docker` folder for development? Not doing so seems to create development/testing coverage gaps by not testing the actual code being used for production/release/deployment

Reply:
Yeah, we can probably get rid of all that now

Reply:
(i mean, we want the docker-compose file and the nginx config - but yes duplicate scripts and dockerfile should go away)

Reply:
Sounds good, if you want to remove those in this PR I can re-approve.